tests/test_ftpfs.py

```diff
@@ -111,82 +82,59 @@ def test_manager(self):
                 raise error_perm('999 foo')
 
 
-@attr('slow')
 class TestFTPFS(FSTestCases, unittest.TestCase):
 
-    def make_fs(self):
-        from fs.ftpfs import FTPFS
-        global ftp_port_offset
-        temp_path = os.path.join(self._temp_dir, text_type(uuid.uuid4()))
-        _ftp_port = ftp_port + ftp_port_offset
-        ftp_port_offset += 1
-
-        os.mkdir(temp_path)
-        env = os.environ.copy()
-
-        server = subprocess.Popen(
-            [
-                sys.executable,
-                os.path.abspath(__file__),
-                temp_path,
-                text_type(_ftp_port)
-            ],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            env=env
-        )
-        server.stdout.readline()
-
-        if _WINDOWS_PLATFORM:
-            # Don't know why this is necessary on Windows
-            time.sleep(0.1)
+    user='user'
+    pasw='1234'
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This does seem simpler than launching numerous ftp servers! Is it faster?

Reply:
Yes ! Running `tests.test_ftpfs` takes 6.5s on my machine :)

Reply:
(except for `test_connection_error` which sometimes takes longer, sometimes not)

Reply:
Is that a socket error? Maybe `socket.settimeout`